builder/vmware/common/driver.go

```diff
@@ -361,13 +356,13 @@ func (d *VmwareDriver) PotentialGuestIP(state multistep.StateBag) ([]string, err
 		dhcpLeasesPath := d.DhcpLeasesPath(device)
 		log.Printf("Trying DHCP leases path: %s", dhcpLeasesPath)
 		if dhcpLeasesPath == "" {
-			return []string{}, fmt.Errorf("no DHCP leases path found for device %s", device)
+			return []string{}, fmt.Errorf("no dhcp leases path found for device %s", device)
```

Comment:
I'm surprised about this change, it feels to me that DHCP should be upper-cased?

Reply:
Resolved in 67054cd.